projects/core/src/error-handling/utils/error-interceptor-utils.spec.ts

```diff
@@ -0,0 +1,103 @@
+import {
+  ErrorInterceptor,
+  ErrorInterceptorPriority,
+} from '../error-interceptors/error-interceptor';
+import {
+  handleInterceptors,
+  sortErrorInterceptors,
+} from './error-interceptor-utils';
+
+describe('error interceptor utils', () => {
```

Comment:
unit tests may need to be adjusted, depending on the agreed strategy of storting